<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommandElement</class>
 <widget class="QWidget" name="CommandElement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>155</width>
    <height>68</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>155</width>
    <height>68</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>68</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QComboBox" name="_cmdComboBox">
        <property name="maxVisibleItems">
         <number>4</number>
        </property>
        <item>
         <property name="text">
          <string>CuCD Continue</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CuCD Dump</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CuCD Real Time</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CuCD End</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CuCD Update</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="_deleteButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="test.qrc">
          <normaloff>:/images/wrong.png</normaloff>:/images/wrong.png</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="_argsLineEdit">
        <property name="placeholderText">
         <string>Cmd args...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="test.qrc"/>
 </resources>
 <connections/>
</ui>
